docs/style-guide.rst

```diff
@@ -194,6 +262,19 @@ No::
     y             = 2;
     long_variable = 3;
 
+Don't include a whitespace in the fallback function:
```

Comment:
One reason I think it's harder to make/cause a "mistake" if there is no space.  People probably shouldn't have single letter functions in the first place, but if they did and accidentally deleted the letter, they would end up with a fallback function [usually the accident should be caught by other means, like the code no longer compiling, but we're trying to keep things "foolproof"].

To an untrained eye `function ()` also looks more like a "mistake" [was a name forgotten?], than `function()` which seems more explicit [they didn't want to give a name, there's no space for a name at all].  There could be arguments for the other way, and if so let's hear them and then try avoid bikeshedding :)